libs/zgui/src/gui.zig

```diff
@@ -3371,6 +3371,88 @@ extern fn zguiGetMouseDragDelta(button: MouseButton, lock_threshold: f32, delta:
 extern fn zguiResetMouseDragDelta(button: MouseButton) void;
 //--------------------------------------------------------------------------------------------------
 //
+// Drag and Drop
+//
+//--------------------------------------------------------------------------------------------------
+pub const DragDropFlags = packed struct(u32) {
+    source_no_preview_tooltip: bool = false,
+    source_no_disable_hover: bool = false,
+    source_no_hold_open_to_others: bool = false,
+    source_allow_null_id: bool = false,
+    source_extern: bool = false,
+    source_auto_expire_payload: bool = false,
+
+    _padding0: u4 = 0,
+
+    accept_before_delivery: bool = false,
+    accept_no_draw_default_rect: bool = false,
+    accept_no_preview_tooltip: bool = false,
+    accept_peek_only: bool = false,
+
+    _padding1: u18 = 0,
+};
+
+const Payload = extern struct {
+    data: *anyopaque = null,
+    data_size: i32 = 0,
+    source_id: u32 = 0,
+    source_parent_id: u32 = 0,
+    data_frame_count: i32 = -1,
```

Comment:
The `i32`s should be `c_int`s and the `u32`s should be `c_uint`, for maximum portability.

I do see that we're already using explicitly 32 bit ints elsewhere in these bindings. I think this is incorrect and they should be changed to match the underlying API (outside of this PR)

Reply:
Alright, I tried to match what's already here and a bunch of stuff was using zig types. Good note on the portability.